src/models/ServerStyleSheet.js

```diff
@@ -39,13 +39,25 @@ class ServerTag implements Tag {
   }
 
   toHTML() {
-    const namesAttr = `${SC_ATTR}="${this.names.join(' ')}"`
-    const localAttr = `${LOCAL_ATTR}="${this.isLocal ? 'true' : 'false'}"`
-    const css = Object.keys(this.components)
-      .map(key => this.components[key].css)
-      .join('')
+    const attrs = [
+      'type="text/css"',
+      `${SC_ATTR}="${this.names.join(' ')}"`,
+      `${LOCAL_ATTR}="${this.isLocal ? 'true' : 'false'}"`,
+    ]
+
+    /* eslint-disable no-underscore-dangle */
```

Comment:
It was complaining about using a comma and I guess there's an eslint rule that all arrow functions have to be one-liners without curlies?